pkgs/tools/security/pass/default.nix

```diff
@@ -1,34 +1,59 @@
-{ stdenv, lib, pkgs, fetchurl, buildEnv
-, coreutils, findutils, gnugrep, gnused, getopt, git, tree, gnupg, openssl
-, which, openssh, procps, qrencode, makeWrapper, pass
-
-, xclip ? null, xdotool ? null, dmenu ? null
-, x11Support ? !stdenv.isDarwin , dmenuSupport ? (x11Support || waylandSupport)
-, waylandSupport ? false, wl-clipboard ? null
-, ydotool ? null, dmenu-wayland ? null
-
-# For backwards-compatibility
-, tombPluginSupport ? false
+{
+  stdenv,
+  lib,
+  pkgs,
+  fetchurl,
+  buildEnv,
+  coreutils,
+  findutils,
+  gnugrep,
+  gnused,
+  getopt,
+  git,
+  tree,
+  gnupg,
+  openssl,
+  which,
+  openssh,
+  procps,
+  qrencode,
+  makeWrapper,
+  pass,
+
+  xclip ? null,
+  xdotool ? null,
+  dmenu ? null,
+  x11Support ? !stdenv.isDarwin,
+  dmenuSupport ? (x11Support || waylandSupport),
+  waylandSupport ? false,
+  wl-clipboard ? null,
+  ydotool ? null,
+  dmenu-wayland ? null,
+
+  # For backwards-compatibility
+  tombPluginSupport ? false,
+
+  extraExtensions ? { },
 }:
 
 assert x11Support -> xclip != null;
 assert waylandSupport -> wl-clipboard != null;
 
 assert dmenuSupport -> x11Support || waylandSupport;
-assert dmenuSupport && x11Support
-  -> dmenu != null && xdotool != null;
-assert dmenuSupport && waylandSupport
-  -> dmenu-wayland != null && ydotool != null;
-
+assert dmenuSupport && x11Support -> dmenu != null && xdotool != null;
+assert dmenuSupport && waylandSupport -> dmenu-wayland != null && ydotool != null;
 
```

Comment:
Which in turn makes all of those useless

Reply:
Removed the unneeded assertions. This is done on a separate commit just for the review: If approved, it will be squashed.